common/test/acceptance/tests/studio/test_studio_settings_certificates.py

```diff
@@ -229,3 +237,71 @@ def test_line_breaks_in_signatory_title(self):
 
         signatory_title = self.certificates_page.get_first_signatory_title()
         self.assertNotEqual([], re.findall(r'<br\s*/?>', signatory_title))
+
+    def test_course_number_in_certificate_details_view(self):
+        """
+        Scenario: Ensure that Course Number is displayed in certificate details view
+
+        Given I have a certificate
+        When I visit certificate details page on studio
+        Then I see Course Number next to Course Name
+        """
+        self.certificates_page.visit()
+        certificate = self.create_and_verify_certificate(
+            "Course Title Override",
+            0,
+            [
+                {
+                    'name': 'Signatory Name',
+                    'title': 'Signatory title with new line character',
```

Comment:
@saleem-latif you can use the Pre defined method here `self.make_signatory_data` , as new line character is not required in this test.